src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1405,7 +1405,7 @@ namespace CoolingPanelSimple {
                 // Now, distribute the radiant energy of all systems to the appropriate surfaces, to people, and the air
                 DistributeCoolingPanelRadGains(state.dataChilledCeilingPanelSimple);
                 // Now "simulate" the system by recalculating the heat balances
-                HeatBalanceSurfaceManager::CalcHeatBalanceOutsideSurf(state.dataConvectionCoefficients, state.files, ZoneNum);
+                HeatBalanceSurfaceManager::CalcHeatBalanceOutsideSurf(state, state.dataConvectionCoefficients, state.files, ZoneNum);
```

Comment:
@mitchute Just skimming the changes and happened to see 3 different `state` arguments here. Is that intended?

Reply:
@mjwitte The DataAirLoop PR (#8210) will clean up a bunch of these. That will also be followed by a couple of other subsequent PRs to just pass ```state``` in everywhere. We were originally trying to keep it as low-level as possible, but it's been causing a merge nightmare for @brianlball, @Myoldmopar, and I. Once this is all done, we may make another pass where we move to only pass the pieces of ```state``` that are actually needed. For now, though, we're just gonna pass it to minimize conflicts.